README.md

```diff
@@ -56,55 +56,81 @@ The following workflow is based on [miniconda/Anaconda](https://docs.conda.io/en
 
 1. Navigate to the root folder of the repository
 1. Execute
-  
-    ```bash
```

Comment:
Please add the deleted space to all code blocks:
* more convenient with regards to only press tab while writing docs
* 3 spaces are very uncommon for indentation